source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -1088,12 +1104,12 @@ bool COpenGLDriver::genericDriverInit(asset::IAssetManager* assMgr)
 	glViewport(0, 0, Params.WindowSize.Width, Params.WindowSize.Height);
 
 	// adjust flat coloring scheme to DirectX version
-	///extGlProvokingVertex(GL_FIRST_VERTEX_CONVENTION_EXT);
+	///COpenGLFunctionTable::glShader.pglProvokingVertex(GL_FIRST_VERTEX_CONVENTION_EXT);
 
 	// We need to reset once more at the beginning of the first rendering.
 	// This fixes problems with intermediate changes to the material during texture load.
     SAuxContext* found = getThreadContext_helper(false);
-	extGlClipControl(GL_UPPER_LEFT, GL_ZERO_TO_ONE); //once set and should never change (engine doesnt track it)
+    COpenGLFunctionTable::glGeneral.pglClipControl(GL_UPPER_LEFT, GL_ZERO_TO_ONE); //once set and should never change (engine doesnt track it)
     glEnable(GL_FRAMEBUFFER_SRGB);//once set and should never change (engine doesnt track it)
     glEnable(GL_TEXTURE_CUBE_MAP_SEAMLESS);//once set and should never change (engine doesnt track it)
     glDepthRange(1.0, 0.0);//once set and should never change (engine doesnt track it)
```

Comment:
all this setup should probably move to context intialization, don't worry about it now we'll tackle it in a separate PR